assets/gaming/splash.json

```diff
@@ -0,0 +1,28 @@
+{
+	"metadata": {
+			"label": "splash",
+			"category": "gaming",
+			"subcategory": "",
+			"website": "https://t.me/splashapp_bot",
+			"description": "Dive in & build SPLASH: the next #1 Telegram Mini-App to earn, mint & trade tokens.",
```

Comment:
please use more neutral description ("the next #1 ...").
"Telegram Mini-App to earn, mint & trade token" would be excellent